### PR TITLE
Exporter refactoring

### PR DESCRIPTION
**Pull request by beteva**
Camel to snake case refactoring. Linting. Python3 compatibility.

### CODE REVIEW

**Comment by IvarsKarpics** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 101
- Added 6
           

Complexity decreasing per file
==============================
+ Command/Exporter.py  -1
+ Command/embl/ExporterClient.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HardwareRepository/HardwareRepository/pullRequest?prid=4495510&bid=15158761)

**Comment by IvarsKarpics** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 101
- Added 5
           

Complexity decreasing per file
==============================
+ Command/Exporter.py  -1
+ Command/embl/ExporterClient.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HardwareRepository/HardwareRepository/pullRequest?prid=4495510&bid=15158761)

File: Command/Exporter.py [hunk 1]
```diff
@@ -88,78 +80,83 @@ def __init__(self, address, port, timeout=3, retries=1):
...
+            self.events_processing_task = gevent.spawn(self.process_events_from_queue)
 
     def _to_python_value(self, value):
+        """Convert exporter value to python one
+        Args:
+            value (str): String from the exporter
+        """
         if value is None:
-            return
+            return value
```
Comment (rhfogh) [inline Command/Exporter.py]: Not important, but 'return' or 'return None' would be more readable.

File: Command/Exporter.py [hunk 2]
```diff
@@ -190,19 +232,18 @@ def __init__(
         ChannelObject.__init__(self, name, username, **kwargs)
 
         self.__exporter = start_exporter(address, port, timeout)
-
-        self.attributeName = attribute_name
+        self.attribute_name = attribute_name
```
Comment (rhfogh) [inline Command/Exporter.py]: attribute_name does not seem to eb set. Should that be '= name'?

File: Command/embl/ExporterClient.py [hunk 3]
```diff
@@ -29,22 +43,30 @@
 
 
 class ExporterClient(StandardClient):
-    def onMessageReceived(self, msg):
-        if msg[:4] == "EVT:":
+    """ExporterClient class"""
+
+    def on_message_received(self, msg):
+        """Act if the message is an event, pass to SatndardClient otherwise.
```
Comment (rhfogh) [inline Command/embl/ExporterClient.py]: Typo?

File: Command/embl/ExporterClient.py [hunk 4]
```diff
@@ -29,22 +43,30 @@
...
+        Args:
+            msg(str): The message.
+        """
+        if msg[:4] == EVENT:
```
Comment (rhfogh) [inline Command/embl/ExporterClient.py]: Less important: better 'if msg.startswith("EVENT")
More important: msg[:4] is only *four* characters long and so can never be equal to "EVENT".

File: Command/Exporter.py (hunk 2 above)
Comment (beteva) [reply Command/Exporter.py]: @rhfogh it is attribute_name, from the __init__ parameters. name and attribute_name are not the same

File: Command/embl/ExporterClient.py (hunk 3 above)
Comment (beteva) [reply Command/embl/ExporterClient.py]: ?

File: Command/embl/ExporterClient.py (hunk 4 above)
Comment (beteva) [reply Command/embl/ExporterClient.py]: EVENT = "EVT:"

File: Command/embl/ExporterClient.py (hunk 3 above)
Comment (rhfogh) [reply Command/embl/ExporterClient.py]: I thought "SatndardClient" was a non-standard spelling ;-)

File: Command/Exporter.py (hunk 2 above)
Comment (rhfogh) [reply Command/Exporter.py]: Ah, you are right. Me bad.

File: Command/embl/ExporterClient.py (hunk 4 above)
Comment (rhfogh) [reply Command/embl/ExporterClient.py]: Again - I cannot read. But it would still be better style to use .startswith

**Comment by rhfogh** [comment]
My requests for change are withdrawn - what is left need not hold up merging. Please go ahead, AFAIAC

File: Command/Exporter.py (hunk 2 above)
Comment (beteva) [reply Command/Exporter.py]: No problem ;-)

File: Command/embl/ExporterClient.py (hunk 3 above)
Comment (beteva) [reply Command/embl/ExporterClient.py]: Absolutely right - highly non-standard. It was my brain blindness which did not see the error.

**Comment by IvarsKarpics** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 101
- Added 5
           

Complexity decreasing per file
==============================
+ Command/Exporter.py  -1
+ Command/embl/ExporterClient.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HardwareRepository/HardwareRepository/pullRequest?prid=4495510&bid=15158761)

**Comment by marcus-oscarsson** [comment]
I'm merging this since you are both approving these changes :)

**Pull request merged**